lib/wiki_course_output.rb

```diff
@@ -62,6 +64,8 @@ def course_details
        | assignment_page = #{@course.wiki_title}
        | slug = #{@course.slug}
        | campaigns = #{@course.campaigns.pluck(:title).join(', ')}
+       | all_instructor_usernames = #{all_instructors_username}
+       | all_instructor_realnames = #{all_instructors_realname}
        | #{@dashboard_url} = yes
```

Comment:
Currently, the template looks something like this:


```
{{course details
 | course_name = Foundations II
 | instructor_username = Health policy
 | instructor_realname = Dorie Apollonio
 | support_staff = Ian (Wiki Ed)
 | subject = Pharmacy
 | start_date = 2024-06-01 00:00:00 UTC
 | end_date = 2024-08-17 23:59:59 UTC
 | institution = UCSF
 | expected_students = 125
 | assignment_page = Wikipedia:Wiki_Ed/UCSF/Foundations_II_(Summer_2024)
 | slug = UCSF/Foundations_II_(Summer_2024)
 | campaigns = Communicating Science, Students in the Health Professions, Summer 2024
 | dashboard.wikiedu.org = yes
}}
```

I think in order for it to work smoothly with additional instructors, instead of combining them all into one parameter, it should include a different parameter for each username after the first. For example, `instructor_username_2` and so on, for the additional instructors after the first. That way, the template can be extended to handle them similarly with wikilinks to their user pages.

Also, let's not pass along the real name of additional instructors. We may at some point remove the real name of the first instructor, as the community has come to value privacy a little more highly relative to editor transparency in recent years, and we may be removing the real name of the first instructor as well at some point.

Reply:
Okay, I will make the necessary changes.